src/train.py

```diff
@@ -172,6 +171,10 @@ def main(args):
         model_args=args.model.model,
         logger=logger,
     )
+    if args.experiment.bucket_name and not args.experiment.debug:
```

Comment:
Looks like this would run only on training loop completion. Personally most of my runs are terminated when I'm fed up with them :P So perhaps this should be (also?) an external python program? or perhaps we can catch the ctrl + c SIGINT like [here](https://www.devdungeon.com/content/python-catch-sigint-ctrl-c)

Reply:
external process probably would be the best, I wanted to have a basic option so we could easily upload existing experiments/catch the ones that trained til the end